build.gradle

```diff
@@ -4,7 +4,8 @@ buildscript {
     }
 
     ext {
-        kotlin_version = "1.5.32"
+        kotlin_version = "1.6.21"
```

Comment:
Please, return back the version of Kotlin. This is not a change for updating this version